packages/playwright-test/src/index.ts

```diff
@@ -463,8 +463,13 @@ export const test = _baseTest.extend<TestFixtures, WorkerFixtures>({
 
     await use(async options => {
       const hook = hookType(testInfo);
-      if (hook)
-        throw new Error(`"context" and "page" fixtures are not supported in ${hook}. Use browser.newContext() instead.`);
+      if (hook) {
+        throw new Error([
+          `"context" and "page" fixtures are not supported in "${hook}" since they are created on a per-test basis.`,
+          `If you would like to reuse a single page between tests, create context manually with browser.newContext(). See https://playwright.dev/docs/test-retries#reuse-single-page-between-tests for details.`,
+          `If you would like to configure your page before each test, do that in beforeEach hook instead.`,
```

Comment:
I like the idea of linking to the docs. It should be noted, it's our first time doing it. I don't recall why, but last time I had a link in an PW Error message, we decided we didn't want the link.

Reply:
We probably should use an aka.ms link instead and figure out a naming schema.
e.g.  https://aka.ms/playwright/reusable-context